File: app/src/main/java/info/firozansari/espressoexamples/fragments/HierarchyFragment.java
package info.firozansari.espressoexamples.fragments;

import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import info.firozansari.espressoexamples.R;

public class HierarchyFragment extends Fragment {

    public static HierarchyFragment newInstance() {
        HierarchyFragment fragment = new HierarchyFragment();
        Bundle bundle = new Bundle();
        fragment.setArguments(bundle);
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_hierarchy, container, false);
    }
}

File: app/src/main/java/info/firozansari/espressoexamples/fragments/HierarchyFragment.kt
package info.firozansari.espressoexamples.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import info.firozansari.espressoexamples.R

class HierarchyFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_hierarchy, container, false)
    }

    companion object {
        fun newInstance(): HierarchyFragment {
            val fragment = HierarchyFragment()
            val bundle = Bundle()
            fragment.arguments = bundle
            return fragment
        }
    }
}